packages/evolution-generator/src/scripts/generate_choices.py

```diff
@@ -74,9 +78,17 @@ def generate_choices(input_file: str, output_file: str):
                 "spread_choices_name": spread_choices_name,
             }
 
-            # Add the 'conditional' field only if it exists
-            if "conditional":
-                choice["conditional"] = conditional
+            # Add conditional field to choice object if it exists
+            if conditional is not None and conditional.endswith("CustomConditional"):
+                # Check to see if the conditional finish with 'CustomConditional'
+                has_custom_conditionals_import = True
+                choice["conditional"] = f"customConditionals.{conditional}"
+            if conditional is not None and not conditional.endswith(
```

Comment:
Le `elif` était correct, juste que `elif condition is not None` aurait été suffisant, pas besoin du and, car si ça terminait par CustomCondition, on serait entré dans le `if`